src/StackExchange.Exceptional.AspNetCore/ExceptionalBuilderExtensions.cs

```diff
@@ -15,12 +16,13 @@ public static class ExceptionalBuilderExtensions
         /// <param name="builder">The <see cref="IApplicationBuilder"/> instance this method extends.</param>
         /// <param name="configureSettings">The action configuring Exceptional settings.</param>
         /// <exception cref="ArgumentNullException"><paramref name="builder"/> or <paramref name="configureSettings"/> is <c>null</c>.</exception>
-        public static IApplicationBuilder UseExceptional(this IApplicationBuilder builder, Action<Settings> configureSettings)
+        public static IApplicationBuilder UseExceptional(this IApplicationBuilder builder, IConfiguration configuration, Action<Settings> configureSettings)
```

Comment:
Same as below, let's see if we can get rid of this argument. Don't focus on this, it's a usability optimization but we'll see what we can do. Right now, this makes it non-optional and that's a weird overload pattern...but I can refactor those.